plugins/contiv/host.go

```diff
@@ -79,18 +80,34 @@ func (s *remoteCNIserver) routeServicesFromHost() *linux_l3.LinuxStaticRoutes_Ro
 	return route
 }
 
-func (s *remoteCNIserver) defaultRouteToHost() *vpp_l3.StaticRoutes_Route {
+func (s *remoteCNIserver) defaultRoute(gwIP string, outIfName string) *vpp_l3.StaticRoutes_Route {
 	route := &vpp_l3.StaticRoutes_Route{
 		DstIpAddr:         "0.0.0.0/0",
-		NextHopAddr:       s.ipam.VEthHostEndIP().String(),
-		OutgoingInterface: s.interconnectAfpacketName(),
-	}
-	if s.useTAPInterfaces {
-		route.OutgoingInterface = tapVPPEndLogicalName
+		NextHopAddr:       gwIP,
+		OutgoingInterface: outIfName,
 	}
 	return route
 }
 
+func (s *remoteCNIserver) routesToHost() []*vpp_l3.StaticRoutes_Route {
+	// list all IPs assigned to host interfaces
+	ips, err := s.getHostLinkIPs()
+	if err != nil {
```

Comment:
logging of the error might be helpful here

Reply:
log added